Tasks/XcodeV5/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -9,7 +9,7 @@
   "loc.group.displayName.devices": "Devices & simulators",
   "loc.group.displayName.advanced": "Advanced",
   "loc.input.label.actions": "Actions",
-  "loc.input.help.actions": "Enter a space-delimited list of actions. Valid options are `build`, `clean`, `test`, `analyze`, and `archive`. For example,`clean build` will run a clean build. See the [xcodebuild man page](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/xcodebuild.1.html).",
+  "loc.input.help.actions": "Enter a space-delimited list of actions. Valid options are `build`, `clean`, `test`, `analyze`, and `archive`. For example,`clean build` will run a clean build.",
```

Comment:
Change this to `Some valid options are ...` since they can change with Xcode versions.

Reply:
Leave out running "man xcodebuild" on the terminal.